src/main/resources/defaultProperties/ExecutionPlatform.properties

```diff
@@ -1,10 +1,10 @@
-###################################################
-##### SHAFT_Engine: ExecutionPlatform.properties
-###################################################
-executionAddress=local
-targetOperatingSystem=Linux-64
-targetBrowserName=GoogleChrome
-headlessExecution=false
-###################################################
-##### END of Properties File
+###################################################
+##### SHAFT_Engine: ExecutionPlatform.properties
+###################################################
+executionAddress=local
+targetOperatingSystem=Linux-64
+targetBrowserName=GoogleChrome
+headlessExecution=false
+###################################################
+##### END of Properties File
 ###################################################
```

Comment:
Looks like something is Wrong in this file line breaks or something... Would you roll it back please.
The rest is perfect! Thank you 🙏🏻

Reply:
Replaced with the one on master and pushed